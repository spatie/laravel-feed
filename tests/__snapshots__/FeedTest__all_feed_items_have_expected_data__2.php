<?php return '<?xml version="1.0" encoding="UTF-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
                        <id>http://localhost/feedBaseUrl/feed2</id>
                                <link href="http://localhost/feedBaseUrl/feed2"></link>
                                <title><![CDATA[Feed 2]]></title>
                                <description>This is feed 2 from the unit tests</description>
                                <updated>2016-01-01T00:00:00+01:00</updated>
                        <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
            <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
            <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
            <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
            <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
            <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
    </feed>
';
