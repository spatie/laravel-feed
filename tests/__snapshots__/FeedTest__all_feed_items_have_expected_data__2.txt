<?xml version="1.0" encoding="UTF-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
                        <id>http://localhost/feedBaseUrl/feed2</id>
                                <link href="http://localhost/feedBaseUrl/feed2"></link>
                                <title><![CDATA[Feed 2]]></title>
                                <description>This is feed 2 from the unit tests</description>
                                <language>en-US</language>
                                <updated>2016-01-01T00:00:00+01:00</updated>
                        <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>2016-01-01T00:00:00+01:00</updated>
        </entry>
    </feed>
